### ISSUE TITLE
whole image annotation

### ISSUE THREAD

**Issue by tianmaliuixing** [paip-2019/challenge#29]
Dear PAIP organizers,
        Recently I had some confusion after visualizing whole tumor, you can see in the images that some whole tumor areas contain white holes, and some do not. Can you explain the difference in detail？
Thanks.
![image](https://user-images.githubusercontent.com/11666931/58860425-acb64980-86de-11e9-9cfd-3354c4e63868.png)

**Comment by PingjunChen** [comment]
There is another case "Traning_phase_1_018.zip", the "Whole" tumor region is smaller than the "Viable" region, which seems unreasonable.  Under this circumstance, the "viable tumor burden" would be larger than 1.0.  

![01_01_0107](https://user-images.githubusercontent.com/32883033/58915247-944e2b00-86ee-11e9-9ade-54544afc521a.png)

I guess for this case, the names of "whole" and "viable" are reversed.

**Comment by hjoonjang** [comment]
Sorry for being late for this. And, so sorry for making those mistakes. We've checked that the problem caused by our side.

The slide @tianmaliuixing mentioned should have been with roughly 4 more big holes, as expected, and the one @PingjunChen mentioned should have been switched obviously.

We have been figuring out how many more the problematic slides are in our train dataset so that we can announce and release the corrected (updated) masks for those erroneous slides. So, till we make an official announcement, please do not use the apparently-problematic slides if you cannot easily correct them like simply switching each other.

Again, we apologize for causing those confusions. We will do our best to resolve it as soon as possible.

**Comment by PingjunChen** [comment]
Thanks, @hjoonjang 
I have tried to process and check the provided training data. Besides above-mentioned problems, I also found several others:
1. `01_01_0111` is rescanned based on the name, but please remove the `scan` from the slide name to keep the consistent naming for all slides.
2. The format of `Phase_1_tumor_burden.csv` and `Phase_2_tumor_burden.csv` is not consistent. `Phase_2_tumor_burden.csv` is more standardized, please adjust `Phase_1_tumor_burden.csv` to have the same format with `Phase_2_tumor_burden.csv`.
3. There are eight cases that their burden values calculated using `viable` and `whole` mask are not consistent with provided ground truth, as can refer #31

**Comment by hjoonjang** [comment]
Thank you for giving more details, @PingjunChen 
We won't forget them when updating the dataset.
